examples/timeseries/timeseries_anomaly_detection.py

```diff
@@ -137,10 +109,10 @@ def create_sequences(values, time_steps=TIME_STEPS):
         output.append(values[i : (i + time_steps)])
     # Convert 2D sequences into 3D as we will be feeding this into
     # a convolutional layer.
-    return np.expand_dims(output, axis=2)
+    return np.stack(output)
```

Comment:
Why this change?

Reply:
It took me a few minutes to figure out what was happening here. Eventually the light went on: "It is stacking the training data". When making the changes this came naturally. The result is the same so fair enough to question the change. I have moved and updated the comments.